### PR TITLE
test

### PR DESCRIPTION
**Pull request by geoffthemedio**
test

### CODE REVIEW

File: GG/CMakeLists.txt [hunk 1]
```diff
@@ -28,7 +28,7 @@ include(UseCodeCoverage)
 include(Util)
 
 set(GIGI_VERSION 0.8.0)
-set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD 20)
```
Comment (geoffthemedio) [inline GG/CMakeLists.txt]: @o01eg it looks like Boost 1.74 has a compile error with C++20 https://github.com/freeorion/freeorion/actions/runs/4945626729/jobs/8842603075#step:15:1121

Is there a way to specify the C++ language version for specific C++ files (GUI.cpp and Texture.cpp)?

Comment (o01eg) [reply GG/CMakeLists.txt]: @geoffthemedio CMake sets `CXX_STANDARD` property per target, so maybe it possible if group those with `OBJECT` target https://cmake.org/cmake/help/latest/command/add_library.html#object-libraries.

Comment (o01eg) [reply GG/CMakeLists.txt]: Something like this https://github.com/o01eg/freeorion/pull/2/commits/5f8846159a6f743d37b6a6e833655aa65b1e8417

Comment (o01eg) [reply GG/CMakeLists.txt]: Looks like it doesn't work: https://github.com/o01eg/freeorion/pull/2/checks

Also Debians including Ubuntu currently provides only 1.74.

Comment (geoffthemedio) [reply GG/CMakeLists.txt]: I'll just have to leave GiGi building with C++17 then

**Pull request merged**